app/routes.js

```diff
@@ -84,4 +84,17 @@ router.get('/visualisations/external-movements-by-:groupField-:chartType', [
   renderVisualisation
 ])
 
+// Safety Diagnostic Tool
+router.get('/safety-diagnostic-tool/', [configureCurrentUrl, function (req, res) {
```

Comment:
Does this need to be within a safetyDiagnosticTool route/section? Given the ticket, I think the intention was to have it on the home screen.